packages/resolvers/vitest.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    testTimeout: 30000,
+    threads: !process.env.CI,
```

Comment:
I wonder if this is still needed? Maybe they fixed vitest on CI?

Reply:
Yep, forgot to remove after fixing CI